packages/elements/src/icon/utils/SpriteLoader.ts

```diff
@@ -0,0 +1,36 @@
+import { SVGLoader } from '@refinitiv-ui/utils/loader.js';
+const spriteUrl = 'https://cdn.ppe.refinitiv.com/public/libs/elf/assets/elf-theme-halo/resources/sprites/icons.svg';
+let spriteCache: Promise<Document> | undefined;
```

Comment:
We patched the request pending not being registered here, but the issue could be better solved where the request is actually made